src/server/report/app.js

```diff
@@ -42,9 +39,6 @@ class App extends LitElement {
 			background-color: #fafafa;
 			grid-area: content;
 		}
-		main > div {
-			padding: 20px;
-		}
```

Comment:
Because the settings bar needs to fill the full width of the right pane, we can't always add padding.